src/_component/open-source/open-source.pug

```diff
@@ -49,42 +57,41 @@
                     | Redux, CSS, Git, JavaScript, SEO, Optimise, React
                 
             .description-part
-                .coloured-dots
-                    .col-1
-                        .row
-                            .blue
-                            p JavaScript 90%
-                        .row
-                            .green
-                            p TypeScript 5.1%
-                    .col-2
-                        .row
-                            .red
-                            p CSS 3.7%
-                        .row
-                            .yellow
-                            p HTML 1.2%
-                .activity
-                    .commits(class=`icon-commits`) Commits: 3,578
-                    .star(class=`icon-stars`) Stars: 392
-                    .commits(class=`icon-forks`) Forks: 77
-                .project-description
-                    p I make the Front-End world a better place every day. I translate geek into English and my passion is to innovate and design for the future.
-                .link-project__wrapper
-                    .icon-project
-                        .icon-project__github(class=`icon-github`)
-                        a.project-link(href="https://github.com/BogdanNegura/my-cv-2") bogdannegura/my-cv
+                input.project-input(type="radio", name="checkbox-input" id="project-input-id")
+                .content-wrapper
+                    .coloured-dots
+                        .col-1
```

Comment:
bad logic, the rows contain columns.
keeping this in mind, as long as it works, I'm fine with it.